recipes/flojoy/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "flojoy" %}
+{% set version = "0.1.15" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flojoy-{{ version }}.tar.gz
+  sha256: 3ce686dc94b5be85c9f75021bc122d0eca92c9cf0d332f8d68d9653c336b1e9e
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10,<3.11
+    - poetry-core
+    - pip
+  run:
+    - python >=3.10.0,<3.11.0
+    - pandas >=2.0.0,<3.0.0
+    - numpy >=1.24.0,<2.0.0
+    - plotly >=5.12.0,<6.0.0
+    - pillow >=10.0.0,<11.0.0
+    - python-dotenv >=1.0.0,<2.0.0
+    - pyyaml >=6.0.0,<7.0.0
+    - requests >=2.29.0,<3.0.0
+    - keyring >=24.2.0,<25.0.0
+    - python-box >=7.0.0,<8.0.0
+    - pydantic >=2.0.0,<3.0.0
+    - huggingface_hub >=0.16.0,<0.17.0
+    - cloudpickle >=2.2.1,<3.0.0
+    - networkx >=3.0.0,<4.0.0
+    - scipy >=1.10.0,<2.0.0
+    - matplotlib-base >=3.6.0,<4.0.0
+    - portalocker >=2.7.0,<3.0.0
+    - qcodes >=0.39.1,<0.40.0
+    - ipykernel
+
+test:
+  imports:
+    - flojoy
+    - flojoy.data_container
+    - flojoy.flojoy_python
+    - flojoy.job_result_builder
+    - flojoy.flojoy_instruction
+    - flojoy.plotly_utils
+    - flojoy.module_scraper
+    - flojoy.job_result_utils
+    - flojoy.data_container
+    - flojoy.utils
+    - flojoy.parameter_types
+    - flojoy.small_memory
+    - flojoy.flojoy_node_venv
+    - flojoy.job_service
+    - flojoy.node_init
+    - flojoy.config
+    - flojoy.node_preflight
+    - flojoy.flojoy_cloud
+    - flojoy.instruments
+    - flojoy.models
+    - flojoy.connection_manager
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://www.flojoy.ai/
+  dev_url: https://github.com/flojoy-ai/python
+  summary: 'Open-source Python client for flojoy visual scripting'
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - itsjoeoui
```

Comment:
@itsjoeoui: do you agree with being a maintainer of this recipe as specified here?

Should we include more people from @flojoy-ai?

Reply:
Yes!

> Will I be able to PR to add more maintainers in the future like this?

You can use [the `@conda-forge-admin, please add user` bot command](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-user-username) to do it easily.

Reply:
Perfect, sounds good 👍